package.json

```diff
@@ -9,7 +9,7 @@
     "test": "cra-webpack-rewired test",
     "storybook": "start-storybook -p 6006 -s public",
     "commit": "git-cz -S",
-    "lint": "npx eslint . --ext .ts,.tsx && npx prettier -c .",
+    "lint": "npx eslint --fix . --ext .ts,.tsx && npx prettier -c .",
```

Comment:
`yarn lint` is dedicated to be executed during the github action, so `--fix` option is useless.
I propose to create `lint:fix` script with enabled `--fix`

also, I recommend using for commits creation the command `yarn commit`, this command also executes linters in fixing mode.

Reply:
I see, thanks for pointing out. I thought `yarn lint` was also executed after a commit through `husky`.

Where can I see the config of the events that are being executed after a certain git hook? I see the .husky folder but I am not used to this setup. I am used to something like this:

```
“husky”: {
  “hooks”: {
    “applypatch-msg”: “echo \”[Husky] applypatch-msg\””,
    “pre-applypatch”: “echo \”[Husky] pre-applypatch\””,
    “post-applypatch”: “echo \”[Husky] post-applypatch\””,
    “pre-commit”: “echo \”[Husky] pre-commit\””,
```